src/app/components/work-item-detail/work-item-detail.component.less

```diff
@@ -13,23 +13,23 @@
   padding: 0;
   overflow-x: hidden;
   overflow-y: scroll;
-  background: $color-pf-white;
+  background: @color-pf-white;
   border-left: 1px solid;
-  border-left-color: (darken($color-pf-white, 20%));
-  -webkit-box-shadow: -4px 0px 2px -2px $color-pf-black-300;
-     -moz-box-shadow: -4px 0px 2px -2px $color-pf-black-300;
-       -o-box-shadow: -4px 0px 2px -2px $color-pf-black-300;
-          box-shadow: -4px 0px 2px -2px $color-pf-black-300;
+  border-left-color: (darken(@color-pf-white, 20%));
+  -webkit-box-shadow: -4px 0px 2px -2px @color-pf-black-300;
```

Comment:
the autoprefixer should handle this in the future - might be worth making a note in the code to remove at a later date